### ISSUE TITLE
Support hotkey for translation of a text from clipboard

### ISSUE THREAD

**Issue by Yanpas** [crow-translate/crow-translate#99]
**Is your feature request related to a problem? Please describe**
I read pdfs in Okular, Crow doesn't recognize selections there.

**Describe the solution you'd like**
4 more global shortcuts, each of which takes the text from clipboard, not from selection

**Comment by Shatur95** [comment]
>  I read pdfs in Okular, Crow doesn't recognize selections there.
Are you sure that the document have OCR?

**Comment by Yanpas** [comment]
Yes, I even see blue rectangles when I select text. Selection is not copied to XOrg secondary buffer (that middle-click one).

**Comment by Shatur95** [comment]
Maybe would be better to just add an option "Use clipboard instead of selection"?

**Comment by Yanpas** [comment]
I think a tick in a tray menu, as well as a distinct setting which both do the same would be sufficient.

But what if we track both events, and translate the latter? E.g.:
- user selects a text
- user presses copy button somewhere
- Crow translates the text from the second event.

Maybe that can become new default, but configurable behaviour.

**Comment by Shatur95** [comment]
Should be done inside [Selection](https://github.com/crow-translate/crow-translate/blob/master/src/selection.h) class.
